hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/BaseVisitor.groovy

```diff
@@ -175,4 +178,19 @@ abstract class BaseVisitor implements Visitor {
   void visitJob(VoldemortBuildPushJob job) {
     visitJob((Job)job);
   }
+
```

Comment:
We still want to have the visitJob(GobblinJob) here in case we develop other visitors in the future